src/commands/game_commands/play.ts

```diff
@@ -1153,6 +1181,28 @@ export default class PlayCommand implements BaseCommand {
             }
         }
 
+        if (
+            gameType === GameType.CLIP &&
+            (!guildPreference.gameOptions.guessTimeout ||
+                guildPreference.gameOptions.guessTimeout >
+                    GuessTimeoutCommand.TIMER_MAX_ACCEPTABLE_CLIP_BEFORE_RESET)
+        ) {
+            await guildPreference.setGuessTimeout(
+                GuessTimeoutCommand.TIMER_DEFAULT_VALUE,
+            );
+        }
+
+        if (
+            gameType !== GameType.CLIP &&
+            guildPreference.isGuessTimeoutSet() &&
```

Comment:
Above you use `!guildPreference.gameOptions.guessTimeout`, here you use `guildPreference.isGuessTimeoutSet()`. Should be consistent here (probably leaning towards the second, since 0 is falsy)

Reply:
Can we also split this conditional by gameType?
```ts
if (gameType === GameType.CLIP) {

    ...
} else{
    ...
}

Would make this a lot easier to read